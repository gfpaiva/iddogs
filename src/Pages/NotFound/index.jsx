import React from 'react';

export default function NotFound() {
  return (
    <h3 className="text-center">
      Página não encontrada
      <span role="img" aria-label="Cachorro">
        🐶
      </span>
    </h3>
  );
}
