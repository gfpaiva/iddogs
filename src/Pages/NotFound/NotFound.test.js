import React from 'react';
import { mount } from 'enzyme';

import NotFound from './index';

describe('<NotFound />', () => {
  it('should mount properly', () => {
    const wrapper = mount(<NotFound />);

    expect(wrapper).toMatchSnapshot();
  });
});
