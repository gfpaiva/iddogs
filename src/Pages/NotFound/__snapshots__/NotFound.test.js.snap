// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should mount properly 1`] = `
<NotFound>
  <h3
    className="text-center"
  >
    Página não encontrada
    <span
      aria-label="Cachorro"
      role="img"
    >
      🐶
    </span>
  </h3>
</NotFound>
`;
